client/dashboard/data/site-backup.ts

```diff
@@ -0,0 +1,47 @@
+import wpcom from 'calypso/lib/wp';
+
+export interface BackupEntry {
+	id: string;
+	started: string;
+	last_updated: string;
+	status:
+		| 'not_started'
+		| 'queued'
+		| 'started'
+		| 'finished'
+		| 'error'
+		| 'error_will_retry'
+		| 'not-accessible'
+		| 'backups-deactivated';
+	period: string;
+	percent: string;
+	discarded: string;
+	is_backup: string;
+	is_scan: string;
+}
+
+/**
+ * Enqueue a backup for a site.
+ * @param siteId - The ID of the site to enqueue a backup for.
+ * @returns A promise that resolves when the backup is enqueued.
+ */
+export function enqueueSiteBackup( siteId: number ): Promise< void > {
+	return wpcom.req.post(
+		{
+			path: `/sites/${ siteId }/rewind/backups/enqueue`,
+			apiNamespace: 'wpcom/v2',
+		},
+		{}
+	);
+}
+
+/**
+ * Fetch the list of backups for a site.
+ * @param siteId - The ID of the site to fetch backups for.
+ * @returns A promise that resolves to the list of backups.
+ */
+export function fetchSiteBackups( siteId: number ): Promise< BackupEntry[] > {
```

Comment:
I think there's an opportunity here to separate these two functions into entity/collection pattern of site-backup.ts and site-backups.ts

Reply:
Even though it's quite premature to do it with just 2 functions, I think it establishes a good pattern for future backup operations and makes the codebase more predictable. Easy enough to refactor now, so I just did :)